app/workers/plos_details.rb

```diff
@@ -20,15 +20,33 @@ def perform(plos_paper)
                  "api_key=#{ENV.fetch('PLOS_API_KEY')}"
 
     detail_resp = Net::HTTP.get_response(URI.parse(detail_url))
+    # did we get a proper response?
+    unless detail_resp.is_a? Net::HTTPSuccess
+      # don't bother parsing broken or empty output
+      sleep 6
```

Comment:
Instead of the `sleep` calls, should we give [sidekiq-throttler](https://github.com/gevans/sidekiq-throttler) a try? Seems cleaner to me and would keep the job from unnecessarily blocking the worker.